apps/condo/domains/acquiring/schema/RegisterMultiPaymentForOneReceiptService.test.js

```diff
@@ -180,14 +180,9 @@ describe('RegisterMultiPaymentForOneReceiptService', () => {
                 } = await makePayer()
                 const receipt = { id: billingReceipts[0].id }
                 const acquiringIntegrationContext = { id: acquiringContext.id }
-                const [newBillingIntegration] = await createTestBillingIntegration(admin)
-
-                // remap acquiring integration with supported billing integrations
-                await updateTestAcquiringIntegration(admin, acquiringIntegration.id, {
-                    supportedBillingIntegrations: { connect: [{ id: newBillingIntegration.id }] },
-                })
+                await createTestBillingIntegration(admin, { group: 'test' })
                 await updateTestAcquiringIntegration(admin, acquiringIntegration.id, {
-                    supportedBillingIntegrations: { disconnect: [{ id: billingIntegration.id }] },
+                    supportedBillingIntegrationsGroup: 'test',
```

Comment:
faker.alphanumeric is more preferable here, since it guarantee you no intersections without other tests